packages/sirius-web/frontend/sirius-web-application/src/modals/update-library/useLibraryVersions.types.ts

```diff
@@ -0,0 +1,64 @@
+/*******************************************************************************
+ * Copyright (c) 2025 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+
+export interface UseLibraryVersionsValue {
+  data: GQLGetLibraryVersionsQueryData | null;
+  loading: boolean;
+}
+
+export interface GQLGetLibraryVersionsQueryData {
+  viewer: GQLViewer;
+}
+
+export interface GQLViewer {
+  library: GQLLibrary | null;
+}
+
+export interface GQLLibrary {
+  namespace: string;
+  name: string;
+  version: string;
+  versions: GQLLibraryVersionConnection;
```

Comment:
`LibraryVersionsConnection` funnily enough, you have not kept the (invalid) name from the backend GraphQL type

Reply:
Done